### PR TITLE
Username / Location update

### PR DESCRIPTION
**Pull request by nikunj1705**

### CODE REVIEW

File: docs/whatsapp/index.md [hunk 1]
```diff
@@ -15,3 +15,8 @@ WhatsApp Business API provides the following features:
 * **Industry Information** - People can now easily identify your business by its category or industry and find out more information (i.e., hours of operation, website, location, description).
 * **Send and Receive** - Enable two-way high value conversations with customers.
 * **Message Templates** - Create message formats that are needed for sending notifications to users.
+* **Username** - Collect WhatsApp username to add authenticity, and improved CTR. Support agents will also have access to the WhatsApp username on our `Live Chat agent` enabling you to personalize communication while interacting with users.
```
Comment (medhaanand) [inline docs/whatsapp/index.md]: Have a look at the usage here

File: docs/whatsapp/index.md [hunk 2]
```diff
@@ -15,3 +15,8 @@ WhatsApp Business API provides the following features:
 * **Industry Information** - People can now easily identify your business by its category or industry and find out more information (i.e., hours of operation, website, location, description).
 * **Send and Receive** - Enable two-way high value conversations with customers.
 * **Message Templates** - Create message formats that are needed for sending notifications to users.
+* **Username** - Collect WhatsApp username to add authenticity, and improved CTR. Support agents will also have access to the WhatsApp username on our `Live Chat agent` enabling you to personalize communication while interacting with users.  
+
+      Tip: We only get the username, when the user sends a message. Username is a part of the payload with user's message.
+      
+* **Location** - Accepting user's location on WhatsApp would now be possible. WhatsApp has an option for users to share their location. And, Haptik can process the location input. This means we can now easily build use cases where a user has to send their location on Whatsapp.
```
Comment (medhaanand) [inline docs/whatsapp/index.md]: why are we sending location information if it is not live?

File: docs/whatsapp/index.md [hunk 3]
```diff
@@ -15,3 +15,8 @@ WhatsApp Business API provides the following features:
 * **Industry Information** - People can now easily identify your business by its category or industry and find out more information (i.e., hours of operation, website, location, description).
 * **Send and Receive** - Enable two-way high value conversations with customers.
 * **Message Templates** - Create message formats that are needed for sending notifications to users.
+* **Username** - Collect WhatsApp username to add authenticity, and improved CTR. Support agents will also have access to the WhatsApp username on our `Live Chat agent` enabling you to personalize communication while interacting with users. Bot builders will also have access to the WhatsApp username.
+
+      Tip: We only get the username, when the user sends a message. Username is a part of the payload with user's message.
```
Comment (tejas-haptik) [inline docs/whatsapp/index.md]: More like a **Note** than a tip.

Also,
**The user name for all the existing user's in the system will be what it was before**

File: docs/whatsapp/index.md (hunk 2 above)
Comment (tejas-haptik) [reply docs/whatsapp/index.md]: **Note for Bot Builder**: Please do not use the Location Picker from HSL Builder to get location from WhatsApp. Send it as a text message to the user asking for location from the user.

File: docs/whatsapp/index.md (hunk 3 above)
Comment (nikunj1705) [reply docs/whatsapp/index.md]: Done, can you approve now?

File: docs/whatsapp/index.md (hunk 1 above)
Comment (nikunj1705) [reply docs/whatsapp/index.md]: Done.

**Pull request merged**